src/measure_theory/outer_measure.lean

```diff
@@ -980,6 +980,9 @@ def extend (s : α) : ℝ≥0∞ := ⨅ h : P s, m s h
 lemma extend_eq {s : α} (h : P s) : extend m s = m s h :=
 by simp [extend, h]
 
+lemma extend_eq_infty {s : α} (h : ¬P s) : extend m s = ∞ :=
```

Comment:
This one should be named `extend_eq_top`: we use `top` in the lemma names even when it is pretty-printed as `∞`

Reply:
```suggestion
lemma extend_eq_top {s : α} (h : ¬P s) : extend m s = ∞ :=
```
Thanks, I was unsure about that